nms/app/views/alarms/components/table/SimpleTable.tsx

```diff
@@ -27,21 +24,20 @@ import Edit from '@material-ui/icons/Edit';
 import FilterList from '@material-ui/icons/FilterList';
 import FirstPage from '@material-ui/icons/FirstPage';
 import LastPage from '@material-ui/icons/LastPage';
-import MaterialTable from '@material-table/core';
+import MaterialTable, {MaterialTableProps} from '@material-table/core';
```

Comment:
We have some strange patterns that aren't really enforced by any check and hence aren't really followed consistently.
I would like to replace this by a fully automated solution so that we don't have to worry about this anymore.